guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
+                        the gain/loss splits from the transaction that reduces
+                        the <guilabel>No of Shares</guilabel>, by putting them
+                        in a separate transaction.
+                      </para>
+                    </note>
+                </listitem>
+              </itemizedlist>
+            </listitem>
+
+            <listitem>
+              <para><guilabel>Scrub</guilabel> &mdash; Scrub only the 
+                highlighted lot. This button is only enabled if a lot is 
+                highlighted. If no buy splits are linked to the highlighted lot,
+                nothing is done. The <guilabel>Scrub</guilabel> button will link
+                any free sell splits to the lot as required until the sum of the
+                sell split amounts equals the sum of the buy split amounts. If
+                the sum of the sell split amounts is greater than the sum of the
+                buy split amounts, a sell split will itself be split into 2
+                splits, with the remaining amount in an unlinked split.
+              </para>
+
+              <note>
+                <para>For both the <guilabel>Scrub Account</guilabel> and the
+                  <guilabel>Scrub</guilabel> buttons:
+                </para>
+
+                <para>The Gain/Loss for each lot is recalculated and a capital
+                  gain transaction is created if needed. The transaction has
+                  1 split for the security account and the other with
+                  account <emphasis>Orphaned Gains-CCC</emphasis> where CCC is
+                  the security currency. The <emphasis>Orphaned Gains-CCC
+                  </emphasis> account is used because <application>&app;
+                  </application> doesn't know which capital gain or loss
+                  account should be used, so the user must change it to the
+                  required income (or expense) gain or loss account.
+                </para>
+              </note>
+            </listitem>
+
+            <listitem>
+              <para><guilabel>Delete</guilabel> &mdash; Delete the highlighted
+                lot. Any linked splits are automatically freed (unlinked). This
+                button is only enabled if a lot is highlighted.
+              </para>
+            </listitem>
+
+            <listitem>
+              <para><guilabel>Close</guilabel> &mdash; Close the <emphasis>
+                Lots in Account SSSS</emphasis> window.
+              </para>
+            </listitem>
+          </itemizedlist>
+
+        </sect4>
+      </sect3>
+      
+      <sect3 id="invest-sellManual">
+        <title>Manual Lot Creation and Linking</title>
+         
+        <para>Before using this feature ensure you are familiar with
+          <xref linkend="invest-sellLimitations" />
+        </para>
+         
+        <para>Here is an example of selling part of a security holding using
```

Comment:
Explain what it is and why you'd use it before launching into an example. It would be better to include the discussion of why one would use this only in cases where one is calculating cap gains based on either LIFO or sale of designated lots (the same thing for securities as far as US personal tax law is concerned) as the scrub function takes care of FIFO and neither works for average-cost.